tests/dataconverter/test_convert.py

```diff
@@ -116,10 +114,10 @@ def test_get_names_of_all_readers():
 def test_cli(caplog, cli_inputs):
     """A test for the convert CLI."""
     runner = CliRunner()
-    result = runner.invoke(dataconverter.convert_cli, cli_inputs)
-    if "--generate-template" in cli_inputs:
+    result = runner.invoke(dataconverter.main_cli, cli_inputs)
+    if "generate-template" in cli_inputs:
         assert result.exit_code == 0
-        assert '"/ENTRY[entry]/NXODD_name/int_value": "None",' in result.stdout
+        assert '"/ENTRY[entry]/NXODD_name/int_value": null,' in result.stdout
```

Comment:
Are all the readers prepared for this change?

Reply:
These changes only affect the CLI. It doesn't change the objects sent to any reader. So they should all work the same as before.